queries/selectors.yml

```
@@ -246,4 +246,10 @@ selectors:
         - method: fqn
           value: aux_autuacao_id
         - method: fqn
-          value: autuacao
\ No newline at end of file
+          value: autuacao
+
+  - name: snapshot_transito
+    definition:
+      union:
+        - method: fqn
+          value: snapshot_transito_autuacao
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)